documentation/asciidoc/computers/getting-started/setting-up.adoc

```diff
@@ -115,7 +115,7 @@ Raspberry Pi models have the following display connectivity:
 |HDMI, audio and composite out via 3.5mm http://en.wikipedia.org/wiki/Phone_connector_(audio)#TRRS_standards[TRRS] jack
 
 |Raspberry Pi 1 Model A+
```

Comment:
Are we really retcon'ing the B+ (and the A+) to be "Raspberry Pi 1, Model B+"..? Is this a thing, did I miss a thing?

Reply:
Good point. I suppose the 1 doesn't _hurt_ anything, and if anything helps identify the device (a la Apple Watch Series 0). But if you feel strongly about it I can remove the 1 since it is not strictly required and might be technically incorrect.